test/OpenTelemetry.Tests.Stress.Logs/Program.cs

```diff
@@ -0,0 +1,53 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Runtime.CompilerServices;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Logs;
+
+namespace OpenTelemetry.Tests.Stress;
+
+public partial class Program
+{
+    private static ILogger logger;
+    private static Payload payload = new Payload();
```

Comment:
same payload intentionally? in practice this would be new new new PayLoads? Or are you doing this to explore the pooling of `LogRecord` to avoid heap alloc?

Reply:
The latter. I think creating payload is happening before the logging API so we don't need to cover it for the SDK optimization purpose.